docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Expo Updates
+sidebar_title: Expo Updates
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for Expo Updates, a protocol for delivering updates to apps running on multiple platforms.
+
+### Conformance
+
+Conforming servers and client-libraries must fulfill all normative requirements. Conformance requirements are described in this document by both descriptive assertions and key words with clearly defined meanings.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in the normative portions of this document are to be interpreted as described in [IETF RFC 2119](https://tools.ietf.org/html/rfc2119). These key words may appear in lowercase and still retain their meaning unless explicitly declared as non‐normative.
+
+A conforming implementation of this protocol may provide additional functionality, but must not where explicitly disallowed or would otherwise result in non‐conformance.
+
+### Overview
+
+An _update_ is defined as a [_manifest_](#manifest-response) together with the assets referenced inside the manifest.
+Expo Updates is a protocol for delivering updates to apps running on multiple platforms.
+
+An app running a conformant Expo Updates client-library will fetch the most recent update from a conformant update server. If the client-library cannot fetch an update or if it already has the most recent update, the client-library will load the update saved in the client-library's cache.
```

Comment:
```suggestion
An app running a conformant Expo Updates client-library will fetch the most recent update from a conformant update server. If the client-library cannot fetch an update or if it already has the most recent update, the client-library will load an update saved in the client-library's cache.
```

Reply:
Hmm, I actually believe `the` is correct here, I can change it to `load the most recent update` if that is clearer?

Reply:
Yes, I think that clarification is necessary for the parsing: `If the client library cannot fetch an update..., the client library will load [the most recent] update saved in the client library's cache.`